pexv2/pd.go

```diff
@@ -0,0 +1,46 @@
+package pexv2
+
+// PresentationDefinition represents a DIF Presentation Definition defined [here].
+// Presentation Definitions are objects that articulate what proofs a Verifier requires
+//
+// [here]: https://identity.foundation/presentation-exchange/#presentation-definition
+type PresentationDefinition struct {
+	ID               string            `json:"id,omitempty"`
+	Name             string            `json:"name,omitempty"`
+	Purpose          string            `json:"purpose,omitempty"`
+	Format           string            `json:"format,omitempty"`
```

Comment:
format is an object no? are you in move fast and break things mode here? if so could you create a ticket to correct in a follow up?

Reply:
on this same note, missing `format` in the `InputDescriptor`

Reply:
also `fields.optional` and `constraints.limit_disclosure` 

I'm guessing all of this was intentionally left out

Reply:
Aye intentionally left out as it's not needed at the moment. Felt this was better than `map[string]any` on the tbdex side

Reply:
@KendallWeihe : removed format for now! https://github.com/TBD54566975/web5-go/commit/9b78e78d5673a39ca34f2e5cfddcb51ad4f9b950